src/components/ArticlePreviewList.jsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import ArticlePreview from './ArticlePreview';
+
+import {
+  articlePreviewList,
+  listItem,
+} from '../styles/articlePreviewList.module.scss';
+
+const ArticlePreviewList = ({
+  previewData,
+  previewImageEdges,
+  authorImageEdges,
+}) => (
+  <div className={articlePreviewList}>
```

Comment:
This could have an aria-label. Also as it is a list, maybe we could use an `<ul>` component even 🤔